src/auth_db/test.rs

```diff
@@ -27,23 +27,23 @@ fn get_bounces_invalid_address() {
 
 #[test]
 fn deserialize_bounce_type() {
-    let bounce_type: BounceType = serde_json::from_value(json!(0)).expect("JSON error");
+    let bounce_type: BounceType = serde_json::from_value(From::from(0)).expect("JSON error");
```

Comment:
`json!` comes from Rocket, which is specific to the service binary so can't be used in the common tests.

Reply:
Is `From::from` here basically converting to a string, and if so would it be clearer to just write the inline string `"0"`?